README.md

```diff
@@ -3,33 +3,76 @@
 __URLs__
 
 * Name all of the parts of the url that you can remember.  In your own words describe what they do.
+```
+Protocol: the connection in which you access the webpage?
+The port: typically 80. dont remember much else
+The path: the file path that of the website.
+the domain: the URL which represents an IP address
+the query: What is being searched for
+anchor tag: link to specific part of page
+
+```
 * Name the pieces of the following urls:
 	* `https://www.google.com/`
+	```
+	Protocol - domain -port
+	```
 	* `https://workbook.galvanize.com/cohorts/41/learning_experiences/367`
+	```
+protocol - domain - port - path
+	```
 	* `http://locahost:5000/animals/puppies?onlycute=1&size=medium#firstpuppy`
+	```
+protocol - domain - port - path - query - anchor?
+	```
 	* `https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_Error`
+	```
+	protocol - domain - port - path - anchor?
+	```
 * Can a server use more than 1 port?
+```
+no
+```
 * Why is https different than http?
-* How does a server interpret the following url's query paramter.  What data structure does it create on the server?
+```
+https is more secure
+```
+* How does a server interpret the following url's query parameter.  What data structure does it create on the server?
 
 ```
 http://locahost:5000/animals?puppies=fido&puppies=max&puppies=moxie
 ```
+```
+3 separate searches for a puppy named fido, max, and moxie
```

Comment:
Good idea,

It really depends on who implemented the server side code.

In general, I would suspect that it would just set `puppies` equal to the last value, `moxie`